components/hab/src/command/pkg.rs

```diff
@@ -571,3 +571,33 @@ pub mod verify {
         Ok(())
     }
 }
+
+pub mod header {
+    use std::path::Path;
+
+    use common::ui::UI;
+    use hcore::crypto::artifact;
+    use std::io::{self, Write};
+
+    use error::Result;
+
+    pub fn start(ui: &mut UI, src: &Path) -> Result<()> {
+        try!(ui.begin(format!("Reading Artifact Header for {}", &src.display())));
```

Comment:
`Artifact` is the term we use elsewhere in the cli help, so we should probably clean that up as well if we just want to use package. (I've got no problem with that, I was just trying to stay consistent with the rest of the cli.